guides/source/action_mailbox_basics.md

```diff
@@ -18,30 +18,32 @@ After reading this guide, you will know:
 What is Action Mailbox?
 -----------------------
 
-Action Mailbox routes incoming emails to controller-like mailboxes for
-processing in Rails. It ships with ingresses for Mailgun, Mandrill, Postmark,
-and SendGrid. You can also handle inbound mails directly via the built-in Exim,
-Postfix, and Qmail ingresses.
+Action Mailbox routes incoming emails to controller-like mailboxes for processing in your Rails application. Action Mailbox is for receiving inbound email, while [Action Mailer](https://guides.rubyonrails.org/action_mailer_basics.html) is for *sending* emails from your Rails application.
 
-The inbound emails are turned into `InboundEmail` records using Active Record
-and feature lifecycle tracking, storage of the original email on cloud storage
-via Active Storage, and responsible data handling with
-on-by-default incineration.
+The inbound emails are routed asynchronously using Active Job to one or several dedicated mailboxes. These emails are turned into `InboundEmail` records using Active Record, which are capable of interacting directly with the rest of your domain model.
 
-These inbound emails are routed asynchronously using Active Job to one or
-several dedicated mailboxes, which are capable of interacting directly
-with the rest of your domain model.
+`InboundEmail` records also provide lifecycle tracking, storage of the original email via Active Storage, and responsible data handling with on-by-default incineration.
+
+Action Mailbox ships with ingresses for external email providers such as Mailgun, Mandrill, Postmark, and SendGrid. You can also handle inbound mails directly via the built-in Exim, Postfix, and Qmail ingresses.
 
 ## Setup
 
-Install migrations needed for `InboundEmail` and ensure Active Storage is set up:
+Action Mailbox has a few moving parts. First you'll run the Action Mailbox installer. Then configure a chosen ingress for handling incoming email. You're then ready to add Action Mailbox routing, create mailboxes and start processing incoming emails.
 
-```bash
-$ bin/rails action_mailbox:install
-$ bin/rails db:migrate
-```
+To start let's install Action Mailbox:
+
+1. `bin/rails action_mailbox:install` - this will create an `application_mailbox.rb` file and copy over migrations.
+2. `bin/rails db:migrate` - this will run the Action Mailbox and Active Storage migrations.
+
+The Action Mailbox table `action_mailbox_inbound_emails` stores incoming message and their processing 'status'.
+
+At this point you can start your rails server and check out `http://localhost:3000/rails/conductor/action_mailbox/inbound_emails`.
```

Comment:
~Should we expand on this to at least mention what you can do there?~
~Just to give some extra context of how that can be used and useful during dev.~

---

I was going to suggest the above, but realized there's a section below about local dev that mentions it and explains better. Maybe we can just link to it from here?